visidata/loaders/pcap.py

```diff
@@ -245,18 +245,18 @@ class IPSheet(Sheet):
     rowtype = 'packets'
     columns = [
         ColumnAttr('timestamp', type=date, fmtstr="%H:%M:%S.%f"),
-        ColumnAttr('ip', width=0),
-        Column('ip_src', width=14, getter=lambda col,row: ipaddress.ip_address(row.ip.src)),
-        Column('ip_dst', width=14, getter=lambda col,row: ipaddress.ip_address(row.ip.dst)),
-        ColumnAttr('ip_hdrlen', 'ip.hl', width=0, helpstr="IPv4 Header Length"),
+        ColumnAttr('ip', type=str, width=0),
+        Column('ip_src', type=str, width=14, getter=lambda col,row: ipaddress.ip_address(row.ip.src)),
+        Column('ip_dst', type=str, width=14, getter=lambda col,row: ipaddress.ip_address(row.ip.dst)),
+        ColumnAttr('ip_hdrlen', 'ip.hl', type=int, width=0, helpstr="IPv4 Header Length"),
         ColumnAttr('ip_proto', 'ip.p', type=lambda v: protocols['ip'].get(v), width=8, helpstr="IPv4 Protocol"),
-        ColumnAttr('ip_id', 'ip.id', width=0, helpstr="IPv4 Identification"),
-        ColumnAttr('ip_rf', 'ip.rf', width=0, helpstr="IPv4 Reserved Flag (Evil Bit)"),
-        ColumnAttr('ip_df', 'ip.df', width=0, helpstr="IPv4 Don't Fragment flag"),
-        ColumnAttr('ip_mf', 'ip.mf', width=0, helpstr="IPv4 More Fragments flag"),
-        ColumnAttr('ip_tos', 'ip.tos', width=0, type=FlagGetter('ip_tos'), helpstr="IPv4 Type of Service"),
-        ColumnAttr('ip_ttl', 'ip.ttl', width=0, helpstr="IPv4 Time To Live"),
-        ColumnAttr('ip_ver', 'ip.v', width=0, helpstr="IPv4 Version"),
+        ColumnAttr('ip_id', 'ip.id', type=int, width=10, helpstr="IPv4 Identification"),
```

Comment:
The typing all makes sense.  (I'm a bit surprised that we need to declare them `str`, but if it works, it's fine).  Do we want to made these unhidden by default though?  That seems cluttered, and you can expose all hidden columns with `gv`.  Though I will defer to you on this, since you are likely the primary consumer of pcap files in VisiData =)

Reply:
I think by default, I'd rather show them. Didn't realize it was all there until I reviewed source and hidden columns. If someone doesn't want those columns, they're welcome to hide them.